src/main/java/world/bentobox/bentobox/managers/HooksManager.java

```diff
@@ -25,10 +25,10 @@ public void registerHook(Hook hook) {
             plugin.log("Hooking with " + hook.getPluginName() + "...");
             if (hook.hook()) {
                 hooks.add(hook);
-            } else {
-                plugin.log("Could not hook with " + hook.getPluginName() + ((hook.getFailureCause() != null) ? " because: " + hook.getFailureCause() : "") + ". Skipping...");
+                return;
             }
         }
+        plugin.logError("Could not hook with " + hook.getPluginName() + ((hook.getFailureCause() != null) ? " because: " + hook.getFailureCause() : "") + ". Skipping...");
```

Comment:
I don't think it should be loaded as an error here: hooks are "optional" things, and I think that failure to load them doesn't do as much harm as the ERROR type tends to make people think.

Reply:
Ok